src/ui/public/number_list/number_list.js

```diff
@@ -26,6 +26,7 @@ uiModules
         self.getUnitName = _.partial($parse($attrs.unit), $scope);
 
         const defaultRange = self.range = parseRange('[0,Infinity)');
+        self.validateOrder = $attrs.validateOrder ? $parse($attrs.validateOrder)($scope) : true;
```

Comment:
What is the advantage over just using `scope: { validateOrder: '=?' }`? That way no extra `$parse` is required and the attribute's presence is easily visible to a consumer of the directive.